scripts/methods.rb

```diff
@@ -33,6 +33,21 @@ def parse_sheets
       data_to_insert = [HEADINGS_INSERT, map_data(row, headings)]
       begin
         x = data_to_insert.transpose.to_h
+        x.each do |key, value|
+          if value != nil and value.to_s.match(/excel::error/i)
```

Comment:
Instead of doing string matches, you can very easily just check is_a? here
```
2.3.1 :026 > y[3]
 => #<Spreadsheet::Excel::Error:0x00000002eea578 @code=29> 
2.3.1 :027 > y[3].class
 => Spreadsheet::Excel::Error 
2.3.1 :028 > 
```

The errors are well defined at https://github.com/zdavatz/spreadsheet/blob/master/lib/spreadsheet/excel/error.rb

The one we are facing is `0x1D` which shows up as 29.

I'd much rather just prefer sending "Unavailable" or `nil`, rather than guessing the value. The issues lies with RBI here not providing correct data.